feat: adding support for HLB CantorMX F411 (#25528)
qmk/qmk_firmware: keyboards/hlb/cantor/readme.md

# HLB CantorMX F411

Project based on original Cantor keyboard (42 key diodeless split keyboard), designed with simplicity in mind. It is inspired on the popular [corne](https://github.com/foostan/crkbd), [ferris](https://github.com/pierrechevalier83/ferris) and [sweep](https://github.com/davidphilipbarr/Sweep) keyboards, aiming to provide a more ergonomic (stronger column stagger) corne-like layout with a simple, easy to assemble and cheap design.
We upgraded blackpill version to STM32F411 8Mhz therefore original firmware maintained by [Diego Palacios](https://github.com/diepala) must be adapted to this hardware.
A case design has beed made to get a full 3DP Cantor project (see below)

* Keyboard Maintainer: [HLB H3lli0n](https://github.com/H3lli0n)
* Hardware Supported: Blackpill STM32F411 8Mhz only
* Hardware Availability: https://github.com/H3lli0n/CantorMx-3DP-case

This firmware is made for EE_HANDS split, to build and flash your firmwares (left and right) please follow the examples below with hand in DFU mode:

> Make+flash example for this split (after setting up your build environment):
>   make hlb/cantor:vial::dfu-util-split-left
>   make hlb/cantor:vial::dfu-util-split-right

Reviewer: ```suggestion
Make example for this keyboard (after setting up your build environment):

    make hlb/cantor:default

Flashing example for this keyboard:

    make hlb/cantor:default:flash
```

Author: I'm adding the template make that will produce a not working split keyboard to be compliant to your rule. Readme file should assist user to compile properly the firmware, anyway

Reviewer: `make hlb/cantor:vial::dfu-util-split-left`

Have you actually tried this command?

# HLB CantorMX F411

Project based on original Cantor keyboard (42 key diodeless split keyboard), designed with simplicity in mind. It is inspired on the popular [corne](https://github.com/foostan/crkbd), [ferris](https://github.com/pierrechevalier83/ferris) and [sweep](https://github.com/davidphilipbarr/Sweep) keyboards, aiming to provide a more ergonomic (stronger column stagger) corne-like layout with a simple, easy to assemble and cheap design.
We upgraded blackpill version to STM32F411 8Mhz therefore original firmware maintained by [Diego Palacios](https://github.com/diepala) must be adapted to this hardware.

> A case design has beed made to get a full 3DP Cantor project (see below)

Reviewer: ```suggestion
A case design has been made to get a full 3DP Cantor project (see below)
```

* Keyboard Maintainer: [HLB H3lli0n](https://github.com/H3lli0n)
* Hardware Supported: Blackpill STM32F411 8Mhz only
* Hardware Availability: https://github.com/H3lli0n/CantorMx-3DP-case

This firmware is made for EE_HANDS split, to build and flash your firmwares (left and right) you have to build firmware for each hand.

> Make example for this keyboard (after setting up your build environment):  
> 
>     make hlb/cantor:default  
> 
> Flashing example for this keyboard:  
> 
>     make hlb/cantor:default:flash  

Reviewer: ```suggestion
Make example for this keyboard (after setting up your build environment):  

    make hlb/cantor:default

Flashing example for this keyboard:

    make hlb/cantor:default:flash
```
